### DISCUSSION TITLE
Non-community data for RDA analysis and Variance Partitioning Analysis

### DISCUSSION THREAD

**Discussion by SuperCC18** [vegandevs/vegan#707] [Q&A]
In the context of Redundancy Analysis (RDA), when examining the relationship between a single response variable, Y, such as Net Primary Productivity (NPP), and two sets of explanatory variables, X and Z, represented as matrices of climate data (e.g., Mean Annual Temperature, Mean Annual Precipitation, Seasonal Variation) and soil data (e.g., Soil Nitrogen, Soil Phosphorus, Soil Carbon), respectively:

Is RDA an appropriate method for analysis if Y is a single-column variable that does not pertain to community data or species abundance?
Alternatively, should multiple regression be considered?
Given the division of explanatory variables into two distinct matrices (climate data and soil data), how would one implement this analysis in R? Specifically, is the following R syntax correct for conducting an RDA and its subsequent VPA?

```r
model_test <- rda(NPP ~ Climate dat + Soil dat, data = full_data)
anova(model_test)
vp <- varpart(NPP, Climate dat,Soil dat)
```

**Comment by gavinsimpson** [comment]
Yes, you can do this. But you could just as well fit a GLM via `glm()` to model NPP.

**Reply by SuperCC18** [reply]
The rda analysis is a dimensional reduction of the response variable Y. But in this case my y is NPP with only one column of data. If I use the glm model, can it regress the two matrices climate data and soil data? I want to know the contribution of climate and soil, not the effect of each climate factor and each soil factor.  So, Is it feasible with GLM to regress against these two matrices simultaneously to assess their collective contribution to NPP, without isolating the effects of individual climate and soil factors?

**Reply by gavinsimpson** [reply]
RDA can't do this either - it is a dimension reduction technique in multivariate $\mathbf{Y}$, not the covariates. RDA has the same requirements and limitations as GLM with respect to covariates. Either way you'll need to have sufficient data to estimate all the coefficients of the model. In the case of a single response variable, RDA is just an `lm()`.
